src/plugins/rel/rel.js

```diff
@@ -96,7 +105,15 @@
                     prev.x = toNumber( ref.getAttribute( "data-x" ) );
                     prev.y = toNumber( ref.getAttribute( "data-y" ) );
                     prev.z = toNumber( ref.getAttribute( "data-z" ) );
+                    prev.rotate.y = toNumber( ref.getAttribute( "data-rotate-y" ) );
+                    prev.rotate.x = toNumber( ref.getAttribute( "data-rotate-x" ) );
+                    prev.rotate.z = toNumber(
+                        ref.getAttribute( "data-rotate-z" ) || ref.getAttribute( "data-rotate" ) );
+
+                    // We don't inherit relatives from relTo slide
                     prev.relative = {};
+                    prev.relative.position = ref.getAttribute( "data-rel-position" ) || "absolute";
+                    prev.relative.rotate = { x:0, y:0, z:0, order: "xyz" };
```

Comment:
Wait, why not? IMO both position and rotation should be relative to relTo slide. Basically it should be used 100% as if it's the previous slide.